go.sum

```diff
@@ -112,6 +112,8 @@ github.com/celo-org/celo-blockchain v1.8.0-beta.1 h1:REgRWX6EitObXwkm+1pVdX6tmQ+
 github.com/celo-org/celo-blockchain v1.8.0-beta.1/go.mod h1:Xwrl4Up8aP/p6TolAiHSTVSlsGtu1N6JZADaJaENYYY=
 github.com/celo-org/celo-blockchain v1.8.0-beta.1.0.20230731110612-df6aac48586a h1:glnOrXXscQ1eVF8igG26/KgXcuOljvFx5S5Ksfl+raY=
 github.com/celo-org/celo-blockchain v1.8.0-beta.1.0.20230731110612-df6aac48586a/go.mod h1:Xwrl4Up8aP/p6TolAiHSTVSlsGtu1N6JZADaJaENYYY=
```

Comment:
will add to the release todos ticket (i'll plan to do it at the very end after all deps are updated, since it's been a bit finicky doing this for rosetta while updating kliento/blockchain deps :( )